SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/devicepins.md

Get Device Pins for Enterprise
------------------------------

> To retrieve information about the device pins active for the enterprise, call [`client.devicePins.listForEnterprise(enterpriseId: String, marker: String?, limit: Int?, direction: OrdeDirection?, fields: [String]?)`][get-device-pins] with the ID of the enterpise. This method will return an iterator object in the completion that you can use to retrieve device pins for the enterprise.

Reviewer: ```suggestion
To retrieve information about the device pins active for the enterprise, call [`client.devicePins.listForEnterprise(enterpriseId: String, marker: String?, limit: Int?, direction: OrdeDirection?, fields: [String]?)`][get-device-pins] with the ID of the enterpise. This method will return an iterator object in the completion, which is used to retrieve device pins for the enterprise.
```
